Sources/XcodeProj/Scheme/XCScheme+LaunchAction.swift

```diff
@@ -42,7 +42,6 @@ public extension XCScheme {
         public var buildConfiguration: String
         public var launchStyle: Style
         public var askForAppToLaunch: Bool?
-        public var pathRunnable: PathRunnable?
```

Comment:
we may need a major release of XcodeProj as this (and the initialiser) would be constitute a breaking change

Alternatively 

```swift
public var pathRunnable: PathRunnable? {
   runnable as? PathRunnable
}
```

Reply:
I was so focused on making sure my changes to PathRunnable were non breaking that I forgot to consider my changes to LaunchAction 🤦🏻‍♂️. I'll add this computed var you suggested.